src/index.html

```diff
@@ -3,10 +3,391 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo_link">
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+          class="logo_img"
+        >
+      </a>
+      <nav class="moyo_nav">
+        <ul class="moyo_nav__list">
+          <li class="moyo_nav__item">
+            <a
+              href="#apple"
+              class="moyo_nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="moyo_nav__item">
+            <a
+              href="#samsung"
+              class="moyo_nav__link">
```

Comment:
be more attentive with code formatting
```suggestion
            <a href="#samsung" class="moyo_nav__link">
```